cpp/open3d/geometry/VoxelGridFactory.cpp

```diff
@@ -143,7 +143,6 @@ std::shared_ptr<VoxelGrid> VoxelGrid::CreateFromTriangleMeshWithinBounds(
                                 box_center, box_half_size, v0, v1, v2)) {
                         Eigen::Vector3i grid_index(widx, hidx, didx);
                         output->AddVoxel(geometry::Voxel(grid_index));
-                        break;
```

Comment:
Add comment here why we should not break (you can simply rephrase your commit message).

Reply:
Done.